client/js/components/button/NavButton.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <!--The attribute data-responsive-menu-helper is used as a hook to append an EventListener in GlobalEventlisteners.js-->
+  <button
+    type="button"
+    class="btn btn--blank font-size-medium"
+    aria-haspopup="true"
+    aria-expanded="false"
```

Comment:
There is no accessible label for the button element. Please use `aria-label` with an appropriate value.

Reply:
done https://github.com/demos-europe/demosplan-core/pull/1389/commits/692686822ba44f7c25e4e7bb57ddbbcf42bcf6af